contracts/contracts/helpers/UpgradeableProxy-08.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: MIT
```

Comment:
I guess after the OZ 4 upgrade, this can be directly imported and used? rather than having a helper file?

Reply:
This file is taken from OZ 3, there was no direct replacement in v4, so I am using a modified version of the v3 contract. `TransparentUpgradeableProxy` in v4 should have worked but it is causing a timeout error in the tests. I wasn't able to find the actual reason why.